gui/default/syncthing/core/syncthingController.js

```diff
@@ -570,7 +611,12 @@ angular.module('syncthing.core')
             // This function should match IsAuthEnabled() in guiconfiguration.go
             var guiCfg = $scope.config && $scope.config.gui;
             if (guiCfg) {
-                return guiCfg.authMode === 'ldap' || (guiCfg.user && guiCfg.password);
+                return (guiCfg.authMode === 'ldap'
+                    || (guiCfg.user && guiCfg.password)
+                    || ((($scope.webauthn || {}).state || {}).credentials || []).some(function (c) {
```

Comment:
Nice trick :grin:

Reply:
:smile: Things you do when you can't use the `?.` operator...